test/components/attribute-picker/attribute-picker.test.js

```diff
@@ -145,5 +145,28 @@ describe('attribute-picker', () => {
 			expect(detail.limit).to.equal(5);
 			expect(component.attributeList.length).to.equal(5);
 		});
+
+		it('should fire the d2l-labs-attribute-picker-text-changed event when inputting text into the text input', async() => {
+			const component = await createComponent(attributeList, { limit: 5 });
+
+			const input = component.shadowRoot.querySelector('input');
+
+			await focusElem(input);
```

Comment:
Not a big deal, but `sendKeysElem` will automatically focus for you, so this line shouldn't be necessary.